playbooks/roles/node/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Assert OS is Ubuntu 18.04
+  assert:
+    that:
+      - ansible_distribution == 'Ubuntu'
+      - ansible_distribution_major_version == '18'
+      - ansible_distribution_version == '18.04'
+    quiet: yes
+
+- name: Check if node is installed
+  command: /bin/bash -c "source /home/{{ ansible_user }}/.nvm/nvm.sh; node -v | cut -c 2-"
+  register: node_check
+  failed_when: "'command not found' in node_check.stdout"
+
+- debug:
+    msg: "[WARNING] Current node version is lower than recommended version {{ node_version }}"
```

Comment:
We should tell the user what to do to get the script to install properly. Example: tell user to uninstall existing version and rerun script, an also to remove the .nvm folder, etc...

Reply:
NOTE: Same comment applies for other warning messages as well, unless we provide an uninstall playbook.

Reply:
Added extra steps to where ever warning was showed.